scripts/establish_conda_env.sh

```diff
@@ -74,6 +74,25 @@ function find_conda_defs ()
   CONDA_DEFS="${CONDA_DEFS//\\//}"
 }
 
+function guess_conda_defs ()
```

Comment:
why are you changing establish_conda_env.sh?

Reply:
this is simply an addition in the script to guess the location of conda if the expected locations are not found. It can reduce the burden on the user every time he/she needs to install the libraries since it automatically tries to find conda and the conda.sh script

Reply:
What does ```which conda ...``` do? (Specifically the ...)

Reply:
sorry i left it there because i was executing another command...i ll remove it. It does not have any effect here